cmd/cmd.go

```diff
@@ -85,7 +85,7 @@ func StartService(cfg common.AresServerConfig, logger bark.Logger, queryLogger b
 		if cfg.Cluster.InstanceName != "" {
 			controllerClientCfg.Headers.Add(clients.InstanceNameHeaderKey, cfg.Cluster.InstanceName)
 		}
-		controllerClient := clients.NewControllerHTTPClient(controllerClientCfg.Host, controllerClientCfg.Port, controllerClientCfg.Headers)
+		controllerClient := clients.NewControllerHTTPClient(*controllerClientCfg)
```

Comment:
actually cluster mode sounds weird for me. For now it's just remote schema not truly a cluster

Reply:
soft agreed.. membership maintained by heart beat is the next thing to follow (in my current plan), that might make the word "cluster" sound better, unless the plan is overriden by something else.